nlp_primitives/tests/test_number_of_words_in_quotes.py

```diff
@@ -0,0 +1,84 @@
+import numpy as np
+import pandas as pd
+import pytest
+
+from ..number_of_words_in_quotes import NumberOfWordsInQuotes
+from .test_utils import PrimitiveT, find_applicable_primitives, valid_dfs
+
+
+class TestNumberOfWordsInQuotes(PrimitiveT):
+    primitive = NumberOfWordsInQuotes
+
+    def test_regular_double_quotes_input(self):
+        x = pd.Series(
+            [
+                'Yes "    "',
+                '"Hello this is a test"',
+                '"Yes" "   "',
+                "",
+                '"Python, java prolog"',
+                '"Python, java prolog" three words here "binary search algorithm"',
+            ]
+        )
+        expected = pd.Series([0, 5, 1, 0, 3, 6]).astype("Int64")
```

Comment:
For these expected calls you can specify the dtype directly when creating the series without having to chain on the `.astype` casting:
```
pd.Series([0, 5, 1, 0, 3, 6], dtype="Int64")
```